recipes/nilearn/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: nilearn
+  version: {{ version }}
+
+source:
+  fn: nilearn-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/n/nilearn/nilearn-{{ version }}.tar.gz
+  md5: 66145792fb2aebf51e221225bd210534
+
+build:
+    number: 0
+    script: python setup.py install
+    features:
+        - vc9   # [win and py27]
```

Comment:
I took this section from the nibabel recipe. nilearn is a pure python package so do we really need this Windows compiler stuff?